packages/dropdowns/src/elements/Dropdown/Dropdown.tsx

```diff
@@ -16,7 +16,7 @@ import { DropdownContext, DROPDOWN_TYPE } from '../../utils/useDropdownContext';
 
 export const REMOVE_ITEM_STATE_TYPE = 'REMOVE_ITEM';
 
-export interface IDropdownProps {
+export interface IDropdownProps extends HTMLAttributes<HTMLSelectElement> {
```

Comment:
This is definitely not a DOM element – the addition of `extends` here is incorrect.